src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,19 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fix file cmdlets context issue when the current context doesn't match with the context of input Track1 object 
```

Comment:
Track1 is internal word, should not use it in public doc.
How about: 
Fix file cmdlets potential context issue when the current context doesn't match with the credential of input Azure File object.

Reply:
Yes this one definitely looks better. Done.